app/src/main/java/com/github/gotify/messages/ListMessageAdapter.java

```diff
@@ -94,8 +105,10 @@ public void onBindViewHolder(@NonNull ViewHolder holder, int position) {
                 .placeholder(R.drawable.ic_placeholder)
                 .into(holder.image);
 
-        holder.setDateTime(message.message.getDate());
-        holder.date.setOnClickListener((ignored) -> holder.switchPreciseDate());
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
+        String timeFormat = prefs.getString(TIME_FORMAT_PREFS_KEY, TIME_FORMAT_RELATIVE);
```

Comment:
Looks good, I've found one little bug. If no time format is configured, and the settings for time format is opened but dismissed, then the absolute time format is shown. The shared preferences get "Relative time" as value, and not the key relative_time_format.

[screencast](https://user-images.githubusercontent.com/14895212/107846859-1016db00-6ddf-11eb-8292-3df7c11a1a40.gif)

Reply:
You're right, I set the default value in the ListPreference as the `time_format_entry_relative` instead of `time_format_value_relative`, my bad.  

Thanks for the thorough review, it should be fine now.